backend/server/data/tasks.json

```diff
@@ -39,20 +40,473 @@
+        "labels": [
+            "mindfulness"
+        ],
+        "states": [
+            {
+                "sadness": 0.0,
+                "lonelyness": 0.0,
+                "sleepyness": 0.0,
+                "anxiousness": 3.0,
+                "stress": 4.0,
+                "anger": 1.0
+            },
+            {
+                "sadness": 0.0,
+                "lonelyness": 0.0,
+                "sleepyness": 0.0,
+                "anxiousness": 4.0,
+                "stress": 2.0,
+                "anger": 0.0
+            }
+        ]
+    },
+
+    "Smile": {
+        "title": "Smile",
+        "description": "Smile at a stranger, greet somebody you wouldn't normally greet. Happiness is contagious.",
+        "labels": [
+            "social",
+            "mindfulness"
+        ],
+        "states": [
+            {
+                "sadness": 4.0,
+                "lonelyness": 1.0,
+                "sleepyness": 0.0,
+                "anxiousness": 0.0,
+                "stress": 0.0,
+                "anger": 0.0
+            },
+            {
+                "sadness": 3.0,
+                "lonelyness": 3.0,
+                "sleepyness": 0.0,
+                "anxiousness": 0.0,
+                "stress": 1.0,
+                "anger": 0.0
+            }
+        ]
+    },
+
+    "Drink a glass of water": {
+        "title": "Drink a glass of water",
+        "description": "Being dehydrated is bad for both your physical and mental health. Drink some H2O!",
+        "labels": [
+            "mindfulness",
+            "workout"
+        ],
+        "states": [
+            {
+                "sadness": 0.0,
+                "lonelyness": 0.0,
+                "sleepyness": 1.0,
+                "anxiousness": 2.0,
+                "stress": 2.0,
+                "anger": 0.0
+            },
+            {
+                "sadness": 0.0,
+                "lonelyness": 0.0,
+                "sleepyness": 2.0,
+                "anxiousness": 3.0,
+                "stress": 1.0,
+                "anger": 0.0
+            }
+        ]
+    },
+
+    "View art": {
+        "title": "View art",
+        "description": "Think about what the artist was feeling when they created the piece of art you are viewing.",
+        "labels": [
+            "mindfulness",
+            "social"
+        ],
+        "states": [
+            {
+                "sadness": 3.0,
+                "lonelyness": 3.0,
+                "sleepyness": 0.0,
+                "anxiousness": 2.0,
+                "stress": 0.0,
+                "anger": 0.0
+            },
+            {
+                "sadness": 0.0,
+                "lonelyness": 2.0,
+                "sleepyness": 0.0,
+                "anxiousness": 3.0,
+                "stress": 0.0,
+                "anger": 0.0
+            }
+        ]
+    },
+
+    "Go to event": { // This would have to retrieve events from the ARC.
```

Comment:
Good that you mention it, but before we can merge it this comment will have to go. Comments will not parse, and cause errors :)